test/IntegrationTests/QuartzTests.cs

```diff
@@ -0,0 +1,48 @@
+// <copyright file="QuartzTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using IntegrationTests.Helpers;
+using Xunit.Abstractions;
+
+namespace IntegrationTests;
+
+public class QuartzTests : TestHelper
+{
+    public QuartzTests(ITestOutputHelper output)
+        : base("Quartz", output)
+    {
+    }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void SubmitsTraces()
+    {
+        using var collector = new MockSpansCollector(Output);
+        SetExporter(collector);
+        collector.Expect("OpenTelemetry.Instrumentation.Quartz");
+
+#if NET462
```

Comment:
```suggestion
#if NETFRAMEWORK
```

Reply:
I do not agree in this case. If we change lowest support netframework version I would like to see it failing.
I am not sure what will be the best solution to handle such scenario. Maybe it will be net472 and we will be able to remove it?